### ISSUE TITLE
AI not priorising which of its converted cities should be reformed first

### ISSUE THREAD

**Issue by LynxAbraxas** [civctp2/civctp2#213]
Issue to keep track of https://github.com/civctp2/civctp2/pull/186#issuecomment-541322065 now that #186 is merged.

**Comment by LynxAbraxas** [comment]
@MartinGuehmann It is not clear to me why the goal settings are given multiple times in `strategies.txt`. Is that intended? E.g. ` GOAL_REFORM` is used 7 times.

**Comment by MartinGuehmann** [comment]
No, the goal settings aren't given multiple times. You have 7 main strategies, and for each strategy you have it only defined once. Each leader uses a certain strategy. That's why they all differ. And then you have minor strategies that overwrite pieces of the main strategy. For instance, STRATEGY_TOO_MANY_CITIES overwrites the given values in the main strategy, if the AI has too many cities. Basically, it stops the AI from getting more cities. For some reason, it also treats GOAL_SETTLE_IN_CITY, even so it does not make the AI having more cities. Maybe the idea was that the AI waits until it has a better government, so that it can use the settlers for cities again.
